scrapypuppeteer/response.py

```diff
@@ -70,14 +77,65 @@ def follow(
             **kwargs,
         )
 
+    def follow_all(
+        self,
+        actions=None,
+        close_page: bool = True,
+        accumulate_meta: bool = False,
+        css=None,
+        xpath=None,
+        **kwargs,
+    ) -> Generator[PuppeteerRequest, None, None]:
+        """
+        Execute actions in the same context but in other browser pages.
+        Only one of `actions`, `css`, or `xpath` must be specified.`
+        Note that original page from which the method was called lasts unaffected.
+
+        :param actions: iterable of PuppeteerActions or selectors
+        :param close_page: whether to close page after request completion
+        :param accumulate_meta: whether to accumulate meta from response
+        :param css: selector
+        :param xpath: selector
+        :return: Iterable[PuppeteerRequest]
+        """
 
-class PuppeteerHtmlResponse(PuppeteerResponse):
+        # Probably, we should ban any PuppeteerAction in `actions` except GoTo
```

Comment:
I agree. I believe we can even forbid any non-navigable actions in PuppeteerRequest if there is no page id.